datacatalog/cloud-client/src/test/java/com/example/datacatalog/CreateFilesetEntryTests.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.datacatalog;
+
+import static org.junit.Assert.assertThat;
+
+import com.google.cloud.datacatalog.EntryGroupName;
+import com.google.cloud.datacatalog.EntryName;
+import com.google.cloud.datacatalog.v1beta1.DataCatalogClient;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import org.hamcrest.CoreMatchers;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/**
+ * Integration (system) tests for {@link CreateFilesetEntry}.
+ */
+@RunWith(JUnit4.class)
+@SuppressWarnings("checkstyle:abbreviationaswordinname")
+public class CreateFilesetEntryTests {
+
+  private ByteArrayOutputStream bout;
+
+  private static String PROJECT_ID = System.getenv().get("GOOGLE_CLOUD_PROJECT");
+  private static String LOCATION = "us-central1";
+  private static String ENTRY_GROUP_ID = "fileset_entry_group";
```

Comment:
these should probably use a randomly generated UUID.

Reply:
Changed, using only the first 8 chars from UUID, since hyphens are not supported on those IDs.